wai-handler-launch/windows.c

```diff
@@ -2,11 +2,7 @@
 #include <shellapi.h>
 #include <stdio.h>
 
-void launch(int port, char *s)
+void launch(char *s)
 {
-    int len = 8 + strlen("http://127.0.0.1:") + strlen(s);
-    char *buff = malloc(len);
-    _snprintf(buff, len, "http://127.0.0.1:%d/%s", port, s);
-    ShellExecute(NULL, "open", buff, NULL, NULL, SW_SHOWNORMAL);
-    free(buff);
+    ShellExecute(NULL, "open", s, NULL, NULL, SW_SHOWNORMAL);
```

Comment:
Wow, that's so much nicer :)

Reply:
I hope that it works. Don't have a windows system for testing in the moment, but I do not see why it shouldn't work.